test/OpenTelemetry.Tests/Metrics/MetricExemplarTests.cs

```diff
@@ -48,6 +50,12 @@ public void TestExemplarsCounter()
         Assert.True(metricPoint.Value.StartTime >= testStartTime);
         Assert.True(metricPoint.Value.EndTime != default);
         var exemplars = GetExemplars(metricPoint.Value);
+
+        // TODO: Modify the test to better test cumulative.
+        // In cumulative where SimpleExemplarReservoir's size is
+        // more than the count of new measurements, it is possible
+        // that the exemplar value is for a measurement that was recorded in the prior
+        // cycle. The current ValidateExemplars() does not handle this case.        
```

Comment:
Removing trailing space...

```suggestion
        // cycle. The current ValidateExemplars() does not handle this case.
```